website/content/en/docs/getting-started/_index.md

```diff
@@ -0,0 +1,236 @@
+
+---
+title: "Getting Started with Karpenter on AWS"
+linkTitle: "Getting Started Guide"
+weight: 10
+menu:
+  main:
+    weight: 10
+---
+
+Karpenter automatically provisions new nodes in response to unschedulable
+pods. Karpenter does this by observing events within the Kubernetes cluster,
+and then sending commands to the underlying cloud provider. 
+
+In this example, the cluster is running on Amazon Web Services (AWS) Elastic
+Kubernetes Service (EKS). Karpenter is designed to be cloud provider agnostic,
+but currently only supports AWS. Contributions are welcomed. 
+
+This guide should take less than 1 hour to complete, and cost less than $0.25.
+Follow the clean-up instructions to reduce any charges.
+
+## Install
+
+Karpenter is installed in clusters with a helm chart.
+
+Karpenter additionally requires IAM Roles for Service Accounts (IRSA). IRSA
+permits Karpenter (within the cluster) to make privileged requests to AWS (as
+the cloud provider). 
+
+### Required Utilities
+
+Install these tools before proceeding:
+
+1. [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2-linux.html)
+2. `kubectl` - [the kubernetes CLI](https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/)
+3. `eksctl` - [the CLI for AWS EKS](https://docs.aws.amazon.com/eks/latest/userguide/eksctl.html)
+
+Login to the AWS CLI with a user that has sufficient privileges to create a
+cluster. 
+
+### Environment Variables
+
+After setting up the tools, set the following environment variables to store
+commonly used values. 
+
+```bash
+export CLUSTER_NAME=$USER-karpenter-demo
+export AWS_DEFAULT_REGION=us-west-2
+AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query Account --output text)
+KARPENTER_VERSION=$(curl -fsSL \
+  https://api.github.com/repos/awslabs/karpenter/releases/latest \
+  | jq -r '.tag_name')
+```
+
+### Create a Cluster
+
+Create a cluster with `eksctl`. The [example configuration](eks-config.yaml) file specifies a basic cluster (name, region), and an IAM role for Karpenter to use. 
+
+```bash
+## This command won't work until a release after v0.2.8 is released
+curl -fsSL  https://raw.githubusercontent.com/awslabs/karpenter/"${KARPENTER_VERSION}"/pkg/cloudprovider/aws/docs/eks-config.yaml \
```

Comment:
Will this still be hosted in Github? Or in the website?

Reply:
sticking with this URL for now